packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -315,4 +315,8 @@ export class FileTreeWidget extends CompressedTreeWidget {
         return inflated;
     }
 
+    protected override getDepthPadding(depth: number): number {
+        // add additional extra depth so file nodes are rendered with padding in relation to the top level root node (to properly differentiate items in the tree)
```

Comment:
I would update the documentation:

```suggestion
        // add additional depth so file nodes are rendered with padding in relation to the top level root node.
```